examples/cassandra/README.md

```diff
@@ -57,103 +71,51 @@ command line tool somewhere in your path.  Please see the
 for installation instructions for your platform.
```

Comment:
This is issue is nothing specific to this document, but important context.

Shortest path to creating a GKE cluster from here:
- Click link
- Click moved to link
- Scroll to hosted solution, click Google Container Engine
- Click "View My Console"
- Click "Create a container cluster"
- Click create (Accepting the defaults)
- Wait for cluster to spin up
- Configure kubectl (assuming alread installed)
  - Click on the cluster name
  - Click show credentials
  - Click fetching credentials for kubectl
  - run the gcloud command in a terminal

It doesn't take that long, but is many hops away from this example and some of the steps weren't immediately obvious to me - how do I configure kubectl once I have a running gke cluster?  How to I create a running GKE cluster from the GKE landing page.

My main concern is that the user shouldn't have to figure out how to do some orthogonal task after starting the example.  A bunch of cut-paste steps is probably ok, but having to switch contexts into "How do I setup a k8s cluster?" is too much to ask.